src/cfg.rs

```diff
@@ -194,4 +246,154 @@ mod tests {
         let outbound = cfg.graph.externals(EdgeDirection::Outgoing);
         assert_eq!(outbound.count(), 1);
     }
+
+    #[test]
+    fn build_cfg_containing_local_unconditional_jump() {
```

Comment:
I don't want to be pedantic, but I'm attaching images of how it was formatted before this PR and another one of the CFG on this branch. I think it helps to understand the cases I've changed:
**old**
![dot_local_unc_jump](https://user-images.githubusercontent.com/4637936/27766218-63ca030e-5ec9-11e7-86d7-9022182d548f.png)
**new**
![new_unc_jump](https://user-images.githubusercontent.com/4637936/27766219-6be9d58c-5ec9-11e7-8d80-a1d3a2df4e03.png)